src/btree.rs

```diff
@@ -1,8 +1,25 @@
 use crate::oram::BUCKET_SIZE;
 
-/// XXX - ceiling method for division but usize may overflow.
+/// Performs ceiling division on usize type.
 pub fn udiv_ceil(a: usize, b: usize) -> usize {
-    (a + b - 1) / b
+    (a as f32 / b as f32).ceil() as usize
+}
+
+/// The size of a complete tree is the next power of two minus one of the number
+/// of items stored.
+pub fn get_complete_tree_size(nb_items: usize, node_capacity: usize) -> usize {
```

Comment:
The size of a tree may be misleading. Maybe prefer its "depth"?
```suggestion
pub fn get_complete_tree_depth(nb_items: usize, node_capacity: usize) -> usize {
```
or even
```rust
pub fn get_tree_depth(nb_items: usize, node_capacity: usize) -> usize {
```

Reply:
No since the depth of a tree would more likely be its height (I guess ?) and the size is the number of nodes in the tree.
I don't recall learning any other naming for such structures but I may be wrong ?